canvas_modules/harness/cypress/integration/canvas/tips.js

```diff
@@ -487,31 +576,34 @@ describe("Test undo redo tooltips for different actions", function() {
 		cy.verifyTipForToolbarItem(".redo-action", "Redo: Edit comment");
 		cy.mouseoutToolbarItem(".redo-action");
 		cy.clickToolbarRedo();
+	});
 
+	it("Test undo redo tooltips are displayed when returned from the actionLabelHandler", function() {
```

Comment:
Please change this to:
"Test undo/redo tooltips are generated correctly for different delete actions: context menu, toolbar and keyboard "